earth_enterprise/BUILD_RHEL_CentOS.md

```diff
@@ -0,0 +1,131 @@
+# GEE Build Setup for CentOS, RHEL 6 and RHEL 7     
+
+## Enable additional repos
+Different yum repositories need to be enabled for different distributions to install several required development tools.  
+
+### CentOS 7 
+```
+sudo yum install epel-release
+```
+
+### RHEL 7  
+
+```
+sudo subscription-manager repos --enable=rhel-7-server-extras-rpms
+sudo subscription-manager repos --enable=rhel-7-server-optional-rpms
+sudo yum install -y epel-release
+```    
+
+### CentOS 6 
+```
+wget http://people.centos.org/tru/devtools-2/devtools-2.repo -O /etc/yum.repos.d/devtools-2.repo
+sudo yum install -y epel-release
+```
+
+### RHEL 6
+```
+sudo subscription-manager repos --enable=rhel-x86_64-workstation-dts-2
+sudo subscription-manager repos --enable=rhel-6-server-optional-rpms
```

Comment:
Looks like we need to install epel for RHEL 6. geos-devel doesn't seem to exist anywhere else